components/eamxx/src/mct_coupling/CMakeLists.txt

```diff
@@ -53,9 +53,49 @@ endif()
 # Create atm lib
 add_library(atm ${ATM_SRC})
 
-# Set f90 compiled modules dir
-set_target_properties(atm PROPERTIES Fortran_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/cmake/atm)
-target_include_directories (atm PUBLIC ${CMAKE_BINARY_DIR}/cmake/atm)
-
 # Link libraries
-target_link_libraries(atm PRIVATE ${SCREAM_LIBS} csm_share)
+# NOTE: when ACE is enable, we can safely discard most of the model
+if (EAMXX_ENABLE_ACE)
```

Comment:
Our CMake build system ought to work without touching this CMakeLists.txt file. The code is a duplicate of what's already in the physics/ace folder.